### DISCUSSION TITLE
Did you commit the new `opcache` folder on purpose?

### DISCUSSION THREAD

**Discussion by leo-unglaub** [phpactor/phpactor#2771] [General]
Hey,
the new `opcache` directory that got introduced in 79fdd0be6a7a13ec9acfc83fabbfbc84472b4320 contains a lot of `.bin` files. Are you sure they got commited on purpose? Or should everyone generate there own?

Greetings
Leo

**Comment by dantleech** [comment]
ERM, no they should not have been comitted 😳

**Reply by dantleech** [reply]
Have removed them in master nowm thanks!
